docs/config.json

```diff
@@ -1349,12 +1349,80 @@
           "slug": "/machine-id/getting-started/"
         },
         {
-          "title": "Platform Guides",
-          "slug": "/machine-id/platform-guides/"
+          "title": "Deploying Machine ID",
+          "slug": "/machine-id/deployment/",
+          "entries": [
+            {
+              "title": "Linux Server",
+              "slug": "/machine-id/deployment/linux/"
+            },
+            {
+              "title": "Google Cloud",
+              "slug": "/machine-id/deployment/gcp/"
+            },
+            {
+              "title": "GitLab",
+              "slug": "/machine-id/deployment/gitlab/"
+            },
+            {
+              "title": "AWS",
+              "slug": "/machine-id/deployment/aws/"
+            },
+            {
+              "title": "Kubernetes",
+              "slug": "/machine-id/deployment/kubernetes/"
+            },
+            {
+              "title": "Jenkins",
+              "slug": "/machine-id/deployment/jenkins/"
+            },
+            {
+              "title": "GitHub Actions",
+              "slug": "/machine-id/deployment/github-actions/"
+            },
+            {
+              "title": "Azure",
+              "slug": "/machine-id/deployment/azure/"
+            },
+            {
+              "title": "CircleCI",
```

Comment:
Any particular reason for this ordering?

I would put the public cloud providers together (Google Cloud, AWS, Azure), and then list the CI/CD platforms together (Jenkins, GHA, CircleCI, GitLab CI)